common/actions/ens.js

```diff
@@ -1,20 +1,7 @@
 // @flow
 
-/*** Resolve ENS name ***/
-export type ResolveEnsNameAction = {
-  type: 'ENS_RESOLVE',
-  payload: string
-};
-
-export function resolveEnsName(name: string): ResolveEnsNameAction {
-  return {
-    type: 'ENS_RESOLVE',
-    payload: name
-  };
-}
-
 /*** Cache ENS address ***/
-export type CacheEnsAddressAction = {
+export type CacheEnsAddress = {
```

Comment:
Any reason for removing this naming convention?

Reply:
I don't mind changing it back, just find it a little redundant to post-fix an action with `action` when its in the `actions` folder

Reply:
The convention so far has been to append Action to the action type definition, however I agree its a bit redundant. We should definitely standardize on one or the other though.